docs/installation.md

```diff
@@ -32,7 +32,6 @@ The installation manifests are located in the [deployments](../deployments) fold
     ```
     $ kubectl apply -f common/custom-resource-definitions.yaml
     ```
-    Note: in Step 3, make sure the Ingress controller starts with the `-enable-custom-resources` [command-line argument](cli-arguments.md).
 
```

Comment:
This step is not optional now because the manifests by default has custom resources enabled. Please update the 31 accordingly. For example:
```
1. Create custom resource definitions for [VirtualServer and VirtualServerRoute](virtualserver-and-virtualserverroute.md) resources:
```

Reply:
Please update https://github.com/nginxinc/kubernetes-ingress/blob/master/docs/virtualserver-and-virtualserverroute.md as well - remove the feature status paragraph.